src/simulate.js

```diff
@@ -9,11 +9,13 @@ var fs = require('fs'),
     simSocket = require('./server/socket'),
     dirs = require('./server/dirs');
 
-var server = cordovaServe();
+var server;
 
 var launchServer = function (opts) {
     opts = opts || {};
 
+    server = cordovaServe();
```

Comment:
We should create a new CordovaServe instance, that internally creates a new express app for us.